envsync.go

```diff
@@ -45,9 +45,11 @@ type Syncer struct {
 // e.g: FOO=bar.
 // FOO is the key and bar is the value.
 //
+// Before the synchronization process will create .bak from target file as backup
 // During the synchronization process, there may be an error.
-// Any key-values that have been synchronized before the error occurred is kept in target.
-// Any key-values that haven't been synchronized because of an error occurred is ignored.
+// If error occured, file target will restored from .bak file that created before.
```

Comment:
If error occurred, target file will be restored from previously created .bak file.